index.js

```diff
@@ -64,7 +65,22 @@ module.exports = (filename, options) => {
 		}
 
 		(async () => {
-			const data = await getStream.buffer(zip.outputStream);
+			let data;
+			if (options.streamOutput) {
+				data = zip.outputStream;
+			} else {
+				try {
+					data = await getStream.buffer(zip.outputStream);
+				} catch (error) {
+					if (error instanceof RangeError && (!error.code || error.code === 'ERR_INVALID_OPT_VALUE' || error.code === 'ERR_OUT_OF_RANGE')) {
```

Comment:
Couldn't you just check `buffer.constants.MAX_LENGTH` upfront instead of relying on some vague error type checking?

Reply:
Yes good idea. I've updated the code to use `getStream's` `maxBuffer` option instead and catch the `MaxBufferError`, which seems to me to be the best option. Unfortunately I've hit a bug with that, so I've also opened sindresorhus/get-stream#40 that fixes the issue I'm having there. If that PR is merged, will it be ok to rely on that fix here or should I look for another option?